### PR TITLE
fixed row vertical orientation

### PR DESCRIPTION
**Pull request by Necromant1k**

### CODE REVIEW

File: site/static/sass/_breakpoints.sass [hunk 1]
```diff
@@ -165,8 +165,9 @@
       .td
         &.text-right-flex
           text-align: right
+          @include flex-direction(column)
```
Comment (patroqueeet) [inline site/static/sass/_breakpoints.sass]: hey, educate me: add small comment what this does.

File: site/static/sass/_breakpoints.sass [hunk 2]
```diff
@@ -165,8 +165,9 @@
...
           @include align-items(flex-end)
-          @include justify-content(flex-end) //align right text desktop
+          @include justify-content(flex-start) //align right text desktop
```
Comment (patroqueeet) [inline site/static/sass/_breakpoints.sass]: is the line ending comment still valid with your changes?

Comment (Necromant1k) [inline site/static/sass/_breakpoints.sass]: oops 
in next pr ill change it 
align-items(flex-end) -> align right text for column direction it just was row direction

File: site/static/sass/_breakpoints.sass (hunk 1 above)
Comment (Necromant1k) [inline site/static/sass/_breakpoints.sass]: it just tells to show content column style

**Pull request merged**